app/konnect/updates.md

```diff
@@ -9,6 +9,11 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## February 2024
+
+**Download button added the documents UI**
+: You can now click the “Download” button to download the markdown file currently being viewed from the documents UI.
```

Comment:
```suggestion
: You can now click the **Download** button to download the markdown file currently being viewed from the documents UI.
```
Is this in Dev Portal? If so, we might want to mention the navigation to get to it just so users know where it is.